sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -121,30 +124,29 @@ internal class RadarApiClient(
         params.putOpt("sdkVersion", RadarUtils.sdkVersion)
         params.putOpt("deviceModel", RadarUtils.deviceModel)
         params.putOpt("deviceOS", RadarUtils.deviceOS)
+        params.putOpt("deviceType", RadarUtils.deviceType)
+        params.putOpt("deviceMake", RadarUtils.deviceMake)
+        params.putOpt("sdkVersion", RadarUtils.sdkVersion)
+        params.putOpt("deviceModel", RadarUtils.deviceModel)
```

Comment:
`sdkVersion` / `deviceModel` / `deviceOS` are already in these params in L124-L126

Reply:
Good catch, fixed